### PR TITLE
Remove extra spaces and correct crosslinks in some documentation

### PR DESCRIPTION
**Pull request by YuriSizov**
Checked the docs for extra spaces between words, and also noticed some cross references to methods and method parameters were incorrect. Not a complete pass, just a few clean-up tweaks.

### CODE REVIEW

File: doc/classes/Bone2D.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
 		<method name="get_default_length" qualifiers="const">
 			<return type="float" />
 			<description>
-				Deprecated. Please use  [code]get_length[/code] instead.
+				Deprecated. Please use [method get_length] instead.
```
Comment (akien-mga) [inline doc/classes/Bone2D.xml:37]: 👀

We can probably remove these and avoid release 4.0 with already deprecated methods with a clear replacement :)

File: doc/classes/EditorNode3DGizmoPlugin.xml [hunk 2]
```diff
@@ -147,7 +147,7 @@
...
+				Override this method to allow selecting subgizmos using mouse drag box selection. Given a [param camera] and [param frustum_planes], this method should return which subgizmos are contained within the frustums. The [param frustum_planes] argument consists of an [code]Array[/code] with all the [code]Plane[/code]s that make up the selection frustum. The returned value should contain a list of unique subgizmo identifiers, these identifiers can have any non-negative value and will be used in other virtual methods like [method _get_subgizmo_transform] or [method _commit_subgizmos]. Called for this plugin's active gizmos.
```
Comment (akien-mga) [inline doc/classes/EditorNode3DGizmoPlugin.xml:150]: Should we replace the Array and Plane by proper class tags?

**Review by akien-mga — APPROVED** [review]
Suggested further improvements, but this looks good as is.

File: doc/classes/EditorNode3DGizmoPlugin.xml (hunk 2 above)
Comment (YuriSizov) [reply doc/classes/EditorNode3DGizmoPlugin.xml:150]: Perhaps 🙃 I think that the method signature should link to those already, so it's not critical.

There were a few others as well, but I didn't want to get too too derailed. I initially only wanted to remove empty spaces, but then noticed the above deprecated methods and took another look.

File: doc/classes/Bone2D.xml (hunk 1 above)
Comment (YuriSizov) [reply doc/classes/Bone2D.xml:37]: It was deprecated in... https://github.com/godotengine/godot/pull/47872. That's probably enough time. I think it was originally a property but now only the methods remain, and they are completely redundant. We can add them to the convertor, which we probably didn't have before.

Comment (YuriSizov) [reply doc/classes/Bone2D.xml:37]: PR'd https://github.com/godotengine/godot/pull/73513

**Pull request merged**